src/algorithms/digi/SiliconPulseDiscretization.cc

```diff
@@ -23,15 +23,15 @@ double SiliconPulseDiscretization::_interpolateOrZero(const TGraph& graph, doubl
                                                       double tMax) const {
   // return 0 if t is outside of tMin - tMax range
   // otherwise, return graph interpolation value
-  if (t < tMin || t > tMax)
+  if (t < tMin || t > tMax) {
     return 0;
-  else {
-    double height = graph.Eval(t, nullptr, "S"); // spline interpolation
-    if (!std::isfinite(height))
-      error("Pulse interpolation returns nan. This happen mostly because there are multiple pulse "
-            "height values at the same time. Did you call PulseCombiner?");
-    return height;
   }
+  double height = graph.Eval(t, nullptr, "S"); // spline         interpolation
+  if (!std::isfinite(height))
+    error("Pulse interpolation         returns nan. This happen mostly because there are multiple "
+          "pulse "
+          "height values at the same time. Did you call PulseCombiner?");
+  return height;
 }
```

Comment:
Reflow strings.
```suggestion
  double height = graph.Eval(t, nullptr, "S"); // spline interpolation
  if (!std::isfinite(height))
    error("Pulse interpolation returns nan. This happen mostly because there are multiple "
          "pulse height values at the same time. Did you call PulseCombiner?");
  return height;
}
```